R/listCaches.R

```diff
@@ -9,6 +9,7 @@
 #' @example
 #' R/examples/example.R
 listCaches = function(cacheSubDir="") {
-	list.files(paste0(getOption("RCACHE.DIR"), cacheSubDir))
+	cacheDirFiles = list.files(paste0(getOption("RCACHE.DIR"), cacheSubDir))
```

Comment:
should this use `getCacheDir` instead of `getOption` here?

Reply:
Yep good idea. It's done now along with a couple other spots.